### DISCUSSION TITLE
Como voce acha que deveria ser a estrutura desse repositório?

### DISCUSSION THREAD

**Discussion by carlaferrarez** [carlaferrarez/learn-github#2] [Ideas]
Discussão sobre a estrutura do repositório

**Comment by carlaferrarez** [comment]
Acredito que deveríamos fazer de acordo com o template da aula

**Reply by carlaferrarez** [reply]
Concordo

**Reply by caiopardal** [reply]
Acredito que poderia seguir um outro formato, apesar de não ter contra pontos fortes para o atual

**Comment by caiopardal** [comment]
Acredito que deveria seguir uma arquitetura MVC
